publication-commons/src/main/java/no/unit/nva/publication/PublicationServiceConfig.java

```diff
@@ -1,19 +1,21 @@
 package no.unit.nva.publication;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import nva.commons.core.Environment;
+import nva.commons.core.JsonUtils;
 
-public final class ServiceEnvironmentConstants {
+public final class PublicationServiceConfig {
```

Comment:
renamed class so that it will not be used somewhere else accidentally. I would have liked the class to not be visible outside the module, but we do not use modules now.

Reply:
A good argument for modules